Document steps to run jest tests. (#38)
CSE-25/quick_start_express: README.md

Reviewer: Mention a point that we must install dependencies before running JS tests. People may miss it!

**Clean-Up**: After testing, you may delete the testing directory if it’s no longer needed.

## Running Tests

> To execute the tests, ensure you are in the root directory of the `quick_start_express` project, then run the following command:
> 
> ```bash
> npm test
> ```

Reviewer 2: ```suggestion
To execute the tests, navigate to the root directory of the `quick_start_express` project where all dependencies are installed, and run the following command:

```bash
npm test
```
```
